lib/command/init.rb

```diff
@@ -70,7 +79,11 @@ def init_aozoraepub3(force = false)
         puts "<bold><red>#{"!!!WARNING!!!".center(70)}</red></bold>".termcolor
         puts "AozoraEpub3の構成ファイルを書き換えます。narouコマンド用に別途新規インストールしておくことをオススメします"
       end
-      aozora_path = ask_aozoraepub3_path
+      if @aozora_dirname then
```

Comment:
__[rubocop]__
* Style/ConditionalAssignment: Use the return of the conditional for variable assignment and comparison.
* Style/MultilineIfThen: Do not use `then` for multi-line `if`. [:ref](https://github.com/bbatsov/ruby-style-guide#no-then)

<!-- Comment by [SideCI](https://sideci.com) -->